bin/add_frontend_server.rb

```diff
@@ -12,28 +12,32 @@
 # channel: ssh channel that you get from open_channel
 # command: Command to execute on the remote host
 # exit_error_string: Error message to throw if the command exits with something other than 1
-def execute_ssh_on_channel(channel, command, exit_error_string)
```

Comment:
Could you please add a comment at this top of the file explaining what it does, with a sample invocation.